vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -57,6 +57,48 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
       - cp /etc/kubernetes/admin.conf /home/{{ IMAGE_USERNAME }}/.kube/config
       - chown {{ IMAGE_USERNAME }}:{{ IMAGE_USERNAME }} /home/{{ IMAGE_USERNAME }}/.kube/config
     files:
+      - path: /usr/local/bin/getconfigs.sh
+        owner: root:root
+        permissions: '0755'
+        content: |
+            #!/bin/bash
+
+            mkdir -p /etc/systemd/system/kubelet.service.d
+            kubeletsvc=$(curl -sSL "https://raw.githubusercontent.com/kubernetes/release/{{ KUBERNETES_BINARIES_CONFIG_VERSION }}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service")
+            if grep -iq ExecStart <<<"${kubeletsvc}"; then
+                echo "${kubeletsvc}" | sed "s:/usr/bin:/usr/local/bin:g" > /etc/systemd/system/kubelet.service
+            else
+            cat <<EOF > /etc/systemd/system/kubelet.service
```

Comment:
I would make it very clear if we failed to download that we are using a backup file, with an obvious log message, to prevent unexpected flakes if the configuration changes, and we are unable to fetch the new one, and did not update the backup.

Reply:
ok. will add that

Reply:
We are no longer using a default config file. 
A check is made on the download, if failure, log the error and exit. On success, no ops.